test/Adapter/Driver/Pdo/StatementTest.php

```diff
@@ -126,4 +127,30 @@ public function testExecute()
         $this->statement->prepare('SELECT 1');
         $this->assertInstanceOf('Zend\Db\Adapter\Driver\Pdo\Result', $this->statement->execute());
     }
+
+    /**
+     * @see https://github.com/zendframework/zend-db/pull/224
+     */
+    public function testExecuteWithSpecialCharInBindParam()
+    {
+        $testSqlite = new TestAsset\SqliteMemoryPdo('CREATE TABLE test (text_ TEXT, text$ TEXT);');
+        $this->statement->setDriver(new Pdo(new Connection($testSqlite)));
+        $this->statement->initialize($testSqlite);
+
+        $this->statement->prepare(
+            'INSERT INTO test (text_, text$) VALUES (:' .
+            md5('text_') .
+            ', :' .
+            md5('text$') .
+            ')'
+        );
```

Comment:
I'd use `sprintf` here to make it more readable:

```php
$this->statement->prepare(sprintf(
    'INSERT INTO test (text_, text$) VALUES (:%s, :%s)',
    md5('text_'),
    md5('text$')
));
```

Reply:
Changed.